### PR TITLE
Changes for v1.2

### PR DESCRIPTION
**Pull request by m-burgoyne**
* Add support for multi location requests
* Add support for a limit query parameter
* Add an annex for recommended aggregation method descriptions for measurementType
* Updates the reference to the CoverageJSON community standard
* Add paragraph to specify that the OpenAPI documentation served by the server as the definitive definition of the query parameters that the EDR server supports
* Fix the time response requirement definition
* Fix duplicate operationids  in OpenAPI examples
* Add support for requesting specific resolutions in the Cube query
* Fix the measurementType examples 
* Change Annex A and Annex B to be normative
* Add support to the Z query parameter for open ranges 
* Add missing Custom dimension definitions
* Correct error in collection bbox definition

### CODE REVIEW

**Review by tomkralidis — APPROVED** [review]

**Comment by tomkralidis** [comment]
> @tomkralidis @m-burgoyne The Generate OpenAPI Bundle workflow needs updating for a more current version of Swagger.

Is there something broken in the current version?

**Comment by chris-little** [comment]
@tomkralidis https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/pull/577/checks but they may not matter. Mentions of node.js too.

**Comment by tomkralidis** [comment]
> @tomkralidis https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/pull/577/checks but they may not matter. Mentions of node.js too. they are warnings rather than failures: `warn deprecated @apidevtools/swagger-cli@4.0.4:`

Ah, ok.  The node.js warnings are a function of the GitHub Actions.  I've updated #581 to fix these warnings.

**Review by chris-little — APPROVED** [review]
Phew!  Well done @m-burgoyne

File: core/standard/sections/annex_history.adoc [hunk 1]
```diff
@@ -25,4 +25,5 @@
 |2022-05-12 |Master branch|Mark Burgoyne | all | Add missing requirements and tests for data_queries metadata
 |2022-06-01 |1.0.1|G.Hobona | all | Final OGC Staff review
 |2023-06-30 |1.1|G.Hobona | all | Final OGC Staff review
+|2024-09-19 |1.2|Mark Burgoyne| all | Changes for v1.2 (link:https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/321[#321], link:https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/398[#398], link:https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/414[#414], link:https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/455[#455], link:https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/479[#479] link:https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/527[#527], link:https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/529[#529], link:https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/560[#560], link:https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/565[#565])
```
Comment (dblodgett-usgs) [inline core/standard/sections/annex_history.adoc:28]: Is this how we want to do history?

File: core/standard/sections/clause_3_references.adoc [hunk 2]
```diff
@@ -4,18 +4,18 @@
...
-* [[[rfc8288,rfc8288]]] Nottingham, M.: IETF RFC 8288, Web Linking, https://tools.ietf.org/rfc/rfc8288.txt[https://tools.ietf.org/rfc/rfc8288.txt]
-* [[[rfc6570,rfc6570]]] Gregorio, J., Fielding, R., Hadley, M., Nottingham, M., Orchard, D.: IETF RFC 6570, URI Template, https://datatracker.ietf.org/doc/html/rfc6570[https://datatracker.ietf.org/doc/html/rfc6570]
+* [[[RFC2616,RFC 2616]]] Fielding, R., Gettys, J., Mogul, J., Frystyk, H., Masinter, L., Leach, P., Berners-Lee, T.: IETF RFC 2616, HTTP/1.1, https://tools.ietf.org/rfc/rfc2616.txt[https://tools.ietf.org/rfc/rfc2616.txt]
+* [[[RFC2818,RFC 2818]]] Rescorla, E.: IETF RFC 2818, HTTP Over TLS, https://tools.ietf.org/rfc/rfc2818.txt[https://tools.ietf.org/rfc/rfc2818.txt]
+* [[[RFC3339,RFC 3339]]] Klyne, G., Newman, C.: IETF RFC 3339, Date and Time on the Internet: Timestamps, https://tools.ietf.org/rfc/rfc3339.txt[https://tools.ietf.org/rfc/rfc3339.txt]
+* [[[RFC3896,RFC 3896]]] Berners-Lee, T., Fielding, R., Masinter, L: IETF RFC 3896, Uniform Resource Identifier (URI): Generic Syntax, https://tools.ietf.org/rfc/rfc3896.txt[https://tools.ietf.org/rfc/rfc3896.txt]
```
Comment (dblodgett-usgs) [inline core/standard/sections/clause_3_references.adoc:10]: This should be 3986 -- don't ask my how I know.

**Review by dblodgett-usgs — APPROVED** [review]
Generally looks great. The "limit" addition is good. May be worth fixing a couple very minor things I noted.

**Review by chris-little — APPROVED** [review]

**Comment by chris-little** [comment]
Agreed to merge at OGC EDR API SWG 2024-10-31 #123

**Comment by lukas-phaf** [comment]
I noticed that the rendering of the (two-column) table in Annex D for measurementType values has a problem, it renders as a 4 column table:
https://docs.ogc.org/DRAFTS/19-086r7.html#measurementType_methods_desc

**Pull request merged**